lib/lotus/utils/path_prefix.rb

```diff
@@ -82,7 +82,7 @@ def relative_join(strings, separator = @separator)
         prefix = @string.gsub(@separator, separator)
 
         self.class.new(
-          Utils::Kernel.Array([prefix, strings]).join(separator),
+          [prefix, strings].join(separator).chomp("/"),
```

Comment:
@jodosha would it make sense to pass the `separator` that is local to `relative_join` to chomp? or actually use `DEFAULT_SEPARATOR`?

Reason I ask is because I'm not familiar with all the use cases of that method, but I wonder if we were handling a path with '#' for separator, if it would make sense to use 'DEFAULT_SEPARATOR` which is '/' and not '#'? 

Hope this makes sense?